src/Storage/Device/S3.php

```diff
@@ -364,7 +364,7 @@ public function read(string $path, int $offset = 0, int $length = null): string
             $end = $offset + $length - 1;
             $this->headers['range'] = "bytes=$offset-$end";
         }
-        $response = $this->call(self::METHOD_GET, $uri);
+        $response = $this->call(self::METHOD_GET, $uri, '', [], false);
```

Comment:
Can we use named arguments instead of passing the default $data and $parameters?

```suggestion
        $response = $this->call(self::METHOD_GET, $uri, decode: false);
```